### DISCUSSION TITLE
Is there a way to mock a FROST-Server with API testing tools like Postman?

### DISCUSSION THREAD

**Discussion by mikedolx** [FraunhoferIOSB/FROST-Server#1087] [Q&A] [answered]
Hello there,

I'd like to know if tere is a way to mock a FROST-Server using tools like Postman? My idea is to have an testing endpoint for developing API clients. 

BR,
Michael

**Comment by hylkevds** [comment, marked as answer]
Mocking the SensorThings API is probably not easy, given the large number of possible URLs that are used. It's probably easier to start a FROST-Server as part of the test, and use a [Batch-request](https://fraunhoferiosb.github.io/FROST-Server/extensions/JsonBatchRequest.html) to import a test dataset.

For Java, dotNet, Python, Go and Rust test environments this could easily be done using [Testcontainers](https://github.com/testcontainers)

**Reply by mikedolx** [reply]
Hi @hylkevds,

thanks for the reply. I already had the idea to self-host a Frost-Server, but currently we don't have any server's here. Eventually i set one up on my local machine using docker.

BR;
Michael
